Tina-ttrimmis/Homework-3/json-homework/json-hw.js

```diff
@@ -0,0 +1,50 @@
+
+/*
+1. Convert the string `jsonString` to a Javascript Object and store it in a variable
+called `flickrObj`
+*/
+let flickrObj = JSON.parse(jsonString);
+console.log(flickrObj);
+
+
+
+/*
+2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+with the date the photo was taken. The first console.log should return the following:
+
+"Edelweiss Air, Airbus A320-214, HB-IJU 2016-02-23T18:48:14Z"
+
+*/
+let i;
+for (i=0; i<flickrObj.items.length; i++) {
```

Comment:
Including spaces around operators increases readability (like `i = 0` and `i < flickrObj...`), so I recommend doing that in your code in the future.